bin/sync_vsc_email_postfix.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+#
+# Copyright 2013-2019 Ghent University
+#
+# This file is part of vsc-administration,
+# originally created by the HPC team of Ghent University (http://ugent.be/hpc/en),
+# with support of Ghent University (http://ugent.be/hpc),
+# the Flemish Supercomputer Centre (VSC) (https://www.vscentrum.be),
+# the Flemish Research Foundation (FWO) (http://www.fwo.be/en)
+# and the Department of Economy, Science and Innovation (EWI) (http://www.ewi-vlaanderen.be/en).
+#
+# https://github.com/hpcugent/vsc-administration
+#
+# All rights reserved.
+#
+"""
+This script creates the list of canonical email adresses for VSC users.
+"""
+from __future__ import print_function
+
+import logging
+
+from vsc.accountpage.sync import Sync
+
+
+class VscPostfixSync(Sync):
+    CLI_OPTIONS = {
+        'postfix_alias_map': ('Location of the postfix canonical map', None, 'store', '/etc/postfix/vsc_canonical'),
```

Comment:
change option name to canonical someting

Reply:
Aargh :D